tests/SprykerSdkTest/AsyncApi/Console/SchemaMessageAddConsoleTest.php

```diff
@@ -70,4 +73,30 @@ public function testAddMessageReturnsErrorCodeAndPrintsErrorMessagesWhenMessageC
         $this->assertSame(AbstractConsole::CODE_ERROR, $commandTester->getStatusCode());
         $this->assertNotEmpty($commandTester->getDisplay());
     }
+
+    /**
+     * @return void
+     */
+    public function testAddMessageReturnsErrorCodeAndPrintsErrorMessagesWhenMessageTypeIsNotPublishAndNotSubscribe(): void
+    {
+        $commandTester = $this->tester->getConsoleTester(SchemaMessageAddConsole::class, false);
+
+        // Assert
+        $this->expectException(InvalidConfigurationException::class);
+        $this->expectExceptionMessage(sprintf(
+            'The option "%s" must either be "%s" or "%s"',
```

Comment:
PLease move that to the Messages class. You used it now in two places already and with a proper method in the Messages class we keep the message itself in one place.